pkg/component/plutono/plutono.go

```diff
@@ -486,7 +525,13 @@ func (p *plutono) getDashboards(ctx context.Context) (map[string]string, error)
 	}
 
 	// this is necessary to prevent hitting configmap size limit.
-	return convertToCompactJSON(dashboards)
+	if dashboards, err := convertToCompactJSON(dashboards); err != nil {
+		return nil, err
+	} else {
+		configMap.Data = dashboards
+	}
```

Comment:
```suggestion
	}
	
	configMap.Data = dashboards
```

Reply:
This is intentional, `dashboards` variable scope is only in if else.